xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -1346,11 +1346,11 @@ void CPeripheralCecAdapter::SetConfigurationFromSettings(void)
     ReadLogicalAddresses(GetSettingInt("standby_devices"), m_configuration.powerOffDevices);
 
   // read the boolean settings
-  m_bUseTVMenuLanguage                 = GetSettingBool("use_tv_menu_language") ? 1 : 0;
+  m_bUseTVMenuLanguage                 = GetSettingBool("use_tv_menu_language") ? true : false;
```

Comment:
If GetSettingBool returns a bool, then the ternary isn't necessary

Reply:
@garbear `clang-tidy` also has a [readability-simplify-boolean-expr](https://clang.llvm.org/extra/clang-tidy/checks/readability-simplify-boolean-expr.html) check which will cleanup expressions like these.

I'm planning to send a follow-up PR for that but I'll add a commit here to cleanup the instances you mentioned.